components/Table.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <section class="m-5">
+    <h2 class="text-lg font-bold">{{ title }}</h2>
+    <table class="table-auto">
+      <thead>
+        <tr>
+          <th v-for="column in columns" :key="column">{{ column }}</th>
+        </tr>
+      </thead>
+      <tbody class="border border-black">
+        <tr v-for="c in numberOfColumns" :key="c" class="border border-black">
+          <td
+            v-for="value in data[c - 1]"
+            :key="value"
+            class="text-center border border-black"
+          >
+            {{ value }}
+          </td>
+        </tr>
+      </tbody>
+    </table>
```

Comment:
Apenas um comentário para reflexão (nada para alterar necessariamente)
As `<table>`s não costumam ser muito responsivas quando pensamos na Web moderna.
Podemos usar aqui aquela noção da abstração de implementação que um componente pode criar. Isto é, o componente chama-se `Table` e isso passa uma noção ao "cliente" -- forma de pensar etc --, mas isso não tem restrições no **como** implementar -- se vai ser com `<table>` ou com várias `<div>`s, tanto faz para o cliente.
Porém, digo que isso é apenas um comentário para reflexão, porque neste momento do MVP responsividade **não é** um atributo de qualidade nosso (talvez apenas por enquanto, mas de fato).

Reply:
É verdade, nem tinha pensado disso. Para falar a verdade mesmo, eu nem lembrava como criar uma tabela (peguei da página do tailwind hahaha). Mas de todo modo, faz sentido isso. Entretanto, vou dar merge já e, quando a responsividade for uma parte necessária para nós, aí mudamos essa implementação :)